integrations/distilgpt2/README.md

```diff
@@ -0,0 +1,41 @@
+#  uAgent DistilGPT2 integrations Examples
+
+### Step 1: Prerequisites
+Before starting, you'll need the following:
+* Python (3.10+ is recommended)
+* Poetry (a packaging and dependency management tool for Python)
+
+### Step 2: Set up .env file
+To run the demo, you need API keys from:
+* HuggingFaceAPI Token
+
+##### HuggingFaceAPI Token
+* Visit HuggingFace.
+* Sign up or log in.
+* Goto Profile -> Settings -> Access Tokens.
+* Copy the old token or create new token.
+
+Once you have token, create a .env file in the distilgpt2-integrations/src directory.
+```bash
+export HUGGIN_FACE_ACCESS_TOKEN="{HuggingFaceAPI Token}"
```

Comment:
Any particular reason for not using their official spelling here at wherever else this is used?
```suggestion
export HUGGING_FACE_ACCESS_TOKEN="{HuggingFaceAPI Token}"
```

Reply:
it was not intentional, it was spelling mistake.